static_src/components/env_var_form.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { generateId } from '../util/element_id';
+import PanelActions from './panel_actions.jsx';
+import Action from './action.jsx';
+
+export default class EnvVarForm extends Component {
```

Comment:
Can this component make use of the existing `Form` component? Or is that component too restrictive for this use case? Let us know, so we can improve it!

Reply:
I didn't want to use use the existing `Form` component for a few reasons:

1. I don't really want to store this state in redux - it's local state that doesn't need to be in redux and I wanted to keep things simple.
2. I have some pending ideas that I'd like to propose for forms, inputs, labels, etc. which are also related to accessibility and I wanted to do that in a separate PR.
3. Because the validation for env vars is (arguably) so simple (only name is required), I haven't even bothered to do any client-side validation since the API picks up the error straight up if you try and submit with an empty name. Of course, I'm not saying we should never do it, but I just felt it was overkill at this stage for what is really just a basic two-input form.

Having said that, if you prefer to use it now I can mash it in, otherwise I'm happy to revisit in a subsequent form proposal 👍